servers/plugins/org.fusesource.ide.server.karaf.core/src/org/fusesource/ide/server/karaf/core/util/KarafUtils.java

```diff
@@ -500,13 +322,73 @@ public static String getBundleSymbolicName(IModule module) throws CoreException
 	 * @return
 	 */
 	public static String getBundleVersion(IModule module, File f) throws CoreException {
```

Comment:
this method is quite complicated. I think that it worth creating test cases for each branch